src/components/Notifications/BehaviorGroup/Wizard/SelectableEventTypeTable.tsx

```diff
@@ -74,33 +76,75 @@ const SelectableEventTypeTableSkeleton: React.FunctionComponent = () => {
   );
 };
 
+const SelectableEventTypeTableRow: React.FunctionComponent<{
+  event: SelectableEventTypeRow;
+  rowIndex: number;
+  onSelect?: (isSelected: boolean, event: EventType) => void;
+  selectionLoading?: boolean;
+}> = ({ event, rowIndex, onSelect, selectionLoading }) => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
+
+  return (
+    <>
+      <Tr key={event.id}>
+        {event.description ? (
+          <Td
+            expand={{
+              rowIndex: rowIndex,
+              isExpanded,
+              onToggle: () => setIsExpanded(!isExpanded),
+              expandId: `expandable-toggle-${event.id}`,
+            }}
+          />
+        ) : (
+          <Td className="pf-u-display-none pf-u-display-block-on-md" />
+        )}
+        <Td
+          modifier={selectionLoading ? 'fitContent' : undefined}
+          noPadding={selectionLoading}
+          select={
+            selectionLoading
+              ? undefined
+              : {
+                  rowIndex: rowIndex,
+                  onSelect: (_event, isSelected) =>
+                    onSelect && onSelect(isSelected, event),
+                  isSelected: event.isSelected,
+                  isDisabled: selectionLoading,
+                }
+          }
+        >
+          {selectionLoading && <Spinner size="sm" />}
+        </Td>
+        <Td>{event.eventTypeDisplayName}</Td>
+        <Td>{event.applicationDisplayName}</Td>
+      </Tr>
+      {event.description && isExpanded && (
+        <Tr>
+          <Td colSpan={4}>
+            <Text className="pf-v5-u-color-200 pf-v5-u-p-sm">
```

Comment:
This is currently aligned under the caret and according to the mocks it should be pushed over to the right to align under the event-type name

Reply:
![Screenshot 2024-06-27 at 11 47 28 AM](https://github.com/RedHatInsights/notifications-frontend/assets/86322239/329878f2-df46-47c4-b5da-9a397b928753)

Reply:
Good catch! :+1: Fixed!
![image](https://github.com/RedHatInsights/notifications-frontend/assets/39098327/8e3ded84-fc3f-4596-8f25-bc48568c710b)
![image](https://github.com/RedHatInsights/notifications-frontend/assets/39098327/d59cd8fd-9c67-4646-b071-ab9e2866d92c)